deploy/ui/form.json

```
@@ -107,26 +107,78 @@
           ]
         },
         {
-          "label": "Admin user",
-          "type": "fieldGroup",
+          "label": "Admin user configuration",
+          "type": "dropDown",
           "required": false,
-          "jsonPath": "$.spec.commonConfig",
           "visible": true,
+          "default": "default",
+          "options": [
+            {
+              "value": "secret",
+              "label": "Secret configuration"
+            },
+            {
+              "value": "default",
+              "label": "Username and Password configuration"
+            }
+          ],
```

I think it would be better to keep default as first, but I'm not sure if this have some effect on order of these options
```suggestion
          "options": [
            {
              "value": "default",
              "label": "Username and Password configuration"
            },
            {
              "value": "secret",
              "label": "Secret configuration"
            }
          ],
```

---

ok